pages/_app.tsx

```diff
@@ -45,6 +45,14 @@ export default function GrouparooWWW(props) {
     if (!existing) storage.setItem(key, value);
   }
 
+  if (router.pathname === "/trial") {
+    return (
+      <>
+        <Component {...props} />{" "}
+      </>
+    );
+  }
+
```

Comment:
I would suggest making a new layout.  We have `main`, and you just created `floating` or something like that.  You can extract the container, error display, and a few other parts from the `trail` page into a new `components/layouts/floating` and use that.  Then, this section becomes something like:

```tsx
const LayoutComponent = router.pathname === "/trial" ? MainLayout : FloatingLayout
return <LayoutComponent><Component {...props} /></LayoutComponent>
```